src/Elliptic/Systems/Xcts/Events/ObserveAdmIntegrals.cpp

```diff
@@ -94,6 +100,11 @@ void local_adm_integrals(
                                       face_inv_spatial_metric(ti::J, ti::L) *
                                       face_extrinsic_curvature(ti::k, ti::l));
 
+    // Compute outward-pointing unit normal.
+    const auto face_r = magnitude(face_inertial_coords);
+    const auto face_unit_normal =
+        tenex::evaluate<ti::I>(face_inertial_coords(ti::I) / face_r());
```

Comment:
You already have the `conformal_face_normal` below. That one has unit magnitude w.r.t. the conformal metric. I see that Ossokine2015 uses `x / r` here. In your test the two are the same because isotropic Schwarzschild has a flat conformal metric. For the binary the two are also the same because we are conformally flat at large distances. So I'd say just use the conformal face normal, or understand what difference this choice makes here. For the volume integral / Gauss law you do have to be careful which normal you are working with here vs the one consistent with the proper volume element.

Reply:
For now, I'll use the `conformal_face_normal` one-form as you suggested (though I still have to transform it into a vector). It's not clear to me which one we want to use for Gauss' law, but we might not even need it if we go with the 1/r expansion approach. I think it's fine to leave it like this for now, and we'll come back to it if/when we need.